docs/content/guide/es/06_variable-replacement.ngdoc

```diff
@@ -89,7 +89,7 @@ angular.module('myApp').controller('Ctrl', ['$scope', function ($scope) {
 y recién después pasarlos como expresión a través del filtro:
 
 <pre>
-{{ 'TRANSLATION_ID' | translate:translationData }}
```

Comment:
Covers everything here:

* First one: `datosDeLaTraduccion` and `translationData` not matching!
* Second one (more seriously): I would not translate internal technical references like `TRANSLATION_ID`, `username` and so on: Given the same tokens, you can easily toggle between languages (in case of differences) and we have a cleaner code style.